lib/Echidna/Config.hs

```diff
@@ -67,8 +67,9 @@ instance FromJSON EConfig where
                 return $ TestConf (\fname -> (== goal fname)  . maybe ResOther classifyRes . view result)
                                   (const psender)
         getWord s d = C Dull . fromIntegral <$> v .:? s .!= (d :: Integer)
-        xc = liftM4 TxConf (getWord "propMaxGas" 8000030) (getWord "testMaxGas" 0xffffffff)
-                           (getWord "maxTimeDelay" 604800)     (getWord "maxBlockDelay" 60480)
+        xc = liftM5 TxConf (getWord "propMaxGas" 8000030) (getWord "testMaxGas" 0xffffffff)
+                           (getWord "maxTimeDelay" 604800)     (getWord "maxBlockDelay" 60480) (getWord "maxValue" 0xffffffff)
```

Comment:
there's no reason to have all these spaces here, maybe we should make this 1 arg per line with applicative instead now?